src/components/BCHAddress.vue

```diff
@@ -33,6 +31,17 @@
           <td>Legacy Address</td>
           <td>{{addressData['legacy']}}</td>
         </tr>
+        <!-- 
```

Comment:
I think we should show the SLP and BCH address. If someone is using a block explorer I think it will be clear enough.

Reply:
done in 4fa770a and 358495f.